docs/src/main/asciidoc/includes/compile-quarkus-quickly.adoc

```diff
@@ -0,0 +1,34 @@
+// tag::quickly-install[]
+The following will build all modules in the Quarkus repository except docs and test modules and install them in your local maven repository with the `999-SNAPSHOT` version:
+
+[source,shell]
+----
+$ ./mvnw clean install -Dquickly \ #<1>
```

Comment:
@ebullient `clean install` is the default goal for the profile `quickly`, so maybe you could leave it out for taciturnity?

Reply:
sure. new PR?

Reply:
Fixed in #26664